packages/maskbook/src/extension/popups/pages/Wallet/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { WalletStartUp } from './components/StartUp'
+import { useWallets } from '@masknet/web3-shared'
+import { Route, Switch } from 'react-router-dom'
+import { DialogRoutes } from '../../index'
+import { ImportWallet } from './components/ImportWallet'
+import { AddDeriveWallet } from './components/AddDeriveWallet'
+
+export default function Wallet() {
+    const wallets = useWallets()
+
+    return wallets.length === 0 ? (
+        <WalletStartUp />
+    ) : (
+        <Switch>
+            <Route path={DialogRoutes.ImportWallet} children={<ImportWallet />} exact />
+            <Route path={DialogRoutes.AddDeriveWallet} children={<AddDeriveWallet />} exact />
```

Comment:
So we're going to support derive wallet? IIRC many codes in our codebase is hard-coded the derive path to `0`. cc @guanbinrui